tests/test_download_image.py

```diff
@@ -25,3 +25,12 @@ def test_download_image(mock_get):
     assert response.status_code == 200
     assert response.headers["Content-Disposition"] == "attachment; filename=overlayed_image.png"
     assert response.headers["Content-Type"] == "image/png"
+
+    # Verify the image content
+    img = Image.open(BytesIO(response.content))
```

Comment:
It seems like the position of the overlayed text is only applied correctly on the downloaded image if I drag an drop the text. Otherwise, it gets applied in 0,0. However, in the website, the initial position of the overlayed text is not 0,0.

Reply:
Thank you for pointing that out. It seems the initial position of the overlayed text is not being correctly captured when the text is not dragged. I'll update the code to ensure the initial position is correctly set based on the default position in the UI. I'll make the necessary changes and update the PR shortly.